ariadne/enums.py

```diff
@@ -154,23 +154,27 @@ def enum_values_in_object_type(
         yield from enum_values_in_field_args(field_name, field)
 
 
-@enum_values_in_types.register(GraphQLInputObjectType)
-def enum_values_in_input_type(
-    type_: GraphQLInputObjectType,
-    field_name,
-) -> Generator[InputFieldWithKeys, None, None]:
-    for input_name, field in type_.fields.items():
+def _get_field_with_keys(field_name, fields):
```

Comment:
We are ordering logic in our code from top to bottom, so this should be declared below `enum_values_in_field_args`.